io.github.masterbillwoo.rise_of_the_penguins_gb.yml

```
@@ -0,0 +1,34 @@
+app-id: io.github.masterbillwoo.rise_of_the_penguins_gb
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
```

this should match the runtime for the base app
```suggestion
runtime-version: '24.08'
```